proto/gz/msgs/dvl_tracking_target.proto

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+syntax = "proto3";
+package gz.msgs;
+option java_package = "com.gz.msgs";
+option java_outer_classname = "DVLTrackingTargetProtos";
+
+/// \ingroup gz.msgs
+/// \interface DVLTrackingTarget
+/// \brief Target used for tracking by a Doppler Velocity Log sensor. Either a bottom ground
+/// or a water mass target. Estimates of target range and position in some reference frame
+/// may be provided.
+
+import "gz/msgs/dvl_kinematic_estimate.proto";
+import "gz/msgs/dvl_range_estimate.proto";
+
+message DVLTrackingTarget
+{
+  enum TargetType
+  {
+    DVL_TARGET_UNSPECIFIED = 0;
+    DVL_TARGET_BOTTOM = 1;
+    DVL_TARGET_WATER_MASS = 2;
+  };
+  /// \brief Type of target used for tracking.
+  TargetType type = 1;
+
+  /// \brief Target range (or distance).
```

Comment:
I'd recommend to mention the units here and in the position fields

Reply:
Done in https://github.com/gazebosim/gz-msgs/pull/317/commits/d27b90f407fedd0d2e34aabdff7b4be2ebfc70b1.